app/controllers/api/ims_imports_controller.rb

```diff
@@ -2,7 +2,9 @@ class Api::ImsImportsController < ApplicationController
   include Concerns::CanvasImsccSupport
 
   def create
-    params[:data][:lti_launches].each do |current_params|
+    lti_launches = params[:data][:lti_launches]
+
+    lti_launches&.each do |current_params|
```

Comment:
When is this nil?

Reply:
If there aren't any lti_launches from the export.

Reply:
For example, with Navigator there isn't always an lti_launch with the course. Sometimes there is, sometimes there isn't.